internal/kafka/test/integration/admin_kafka_suspension_test.go

```diff
@@ -109,18 +110,26 @@ func TestAdminKafka_KafkaSuspension(t *testing.T) {
 	}
 	g.Expect(err).ToNot(gomega.HaveOccurred(), "failed to suspend Kafka instance")
 	g.Expect(privateKafkaReq.Status).To(gomega.Equal(kafkaconstants.KafkaRequestStatusSuspending.String()))
+	err = common.WaitForMetricToBePresent(h, t, metrics.KafkaRequestsStatusCount, kafkaconstants.KafkaRequestStatusSuspending.String())
+	g.Expect(err).ToNot(gomega.HaveOccurred())
 
 	// 'suspending' Kafkas should be updated to 'suspended' by the mock fleetshard sync
 	_, err = common.WaitForKafkaToReachStatus(publicClientCtx, kafkatest.TestServices.DBFactory, publicClient, publicKafkaReq.Id, kafkaconstants.KafkaRequestStatusSuspended)
 	g.Expect(err).ToNot(gomega.HaveOccurred(), "kafka failed to reach %q status", kafkaconstants.KafkaRequestStatusSuspended)
 
+	err = common.WaitForMetricToBePresent(h, t, metrics.KafkaRequestsStatusCount, kafkaconstants.KafkaRequestStatusSuspended.String())
+	g.Expect(err).ToNot(gomega.HaveOccurred())
+
 	privateKafkaReq, resp, err = adminClient.DefaultApi.GetKafkaById(adminClientCtx, privateKafkaReq.Id)
 	if resp != nil {
 		resp.Body.Close()
 	}
 	g.Expect(err).ToNot(gomega.HaveOccurred(), "failed to get Kafka instance from admin get endpoint")
 	g.Expect(privateKafkaReq.Status).To(gomega.Equal(kafkaconstants.KafkaRequestStatusSuspended.String()))
 
+	err = common.WaitForMetricToBePresent(h, t, metrics.KafkaRequestsStatusCount, kafkaconstants.KafkaRequestStatusResuming.String())
+	g.Expect(err).ToNot(gomega.HaveOccurred())
```

Comment:
This check has to go in line https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/e736b24f36667f564497721800dec67cef5a2a92/internal/kafka/test/integration/admin_kafka_suspension_test.go#L152 after the Kafka has been set to `resuming` via the admin call: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/e736b24f36667f564497721800dec67cef5a2a92/internal/kafka/test/integration/admin_kafka_suspension_test.go#L146-L151

At the moment, the test is passing since the metric is always reported for all the statuses but the metric value is `0`.
We want to make sure that we assert against the reported metric value as well as state on the other comment